src/dune_file_watcher/dune_file_watcher.ml

```diff
@@ -167,23 +167,6 @@ type t =
   ; ignored_files : (string, unit) Table.t
   }
 
-let exclude_patterns =
-  [ {|/_opam|}
-  ; {|/_esy|}
-  ; {|/\..+|}
-  ; {|~$|}
-  ; {|/#[^#]*#$|}
-  ; {|4913|} (* https://github.com/neovim/neovim/issues/3460 *)
```

Comment:
By the way, I think we need a workaround for this "4913" somewhere. It's created by all vim implementations and there's no way to put it another directory like with swap files for example.

Reply:
Agreed.